### PR TITLE
[QA-239] Update selenium-java version to 3.0.1

### PR DESCRIPTION
**Pull request by gbecares-stratio**
Main Changes:
- Update Selenium-Java version to 3.0.1

Needed:
- Use new selenium hub image: "selenium/hub:3.0.1" named as "selenium301"
https://stratio.atlassian.net/browse/SYSITP-1366 [TO DO]

### CODE REVIEW

**Comment by adoblas** [comment]
@gbecares this PR breakes the retrocompatibility?? 

I guess that when Stratio modules update the BDT dependency (0.5.0-SNAPSHOT), they also MUST change their CHROME service version??

**Comment by witokondoria** [comment]
Completely unrelated changes (guava, curator and so) must be left out of this pr

**Comment by gbecares** [comment]
@witokondoria 

- Since Selenium 3.2 version is recommended to use Guava 21.0
https://github.com/SeleniumHQ/selenium/blob/master/java/CHANGELOG#L31

- Due to this change, cassandra-java-driver and curator versions have been upgraded to make it compatible with them.

**Comment by witokondoria** [comment]
"Users are recommended to update too" As the problem here is a dependency resolution conflict, exclude guava from the selenium dependency and let guava and curator stay still (for now).

**Comment by adoblas** [comment]
ping

**Comment by gbecares** [comment]
Finally Selenium upgraded to 3.0.1.

Dependencies as Guava, Cassandra driver and Curator have to be upgraded in next versions too.

Do not merged until this ticket is DONE: https://stratio.atlassian.net/browse/SYSITP-1366 
and container's name is changed also: https://github.com/gbecares/bdt/blob/f9de61705e7f9d5a8674c73970b8ace8a51fa412/Jenkinsfile#L51

File: Jenkinsfile [hunk 1]
```diff
@@ -44,19 +44,18 @@ hose {
...
            'sleep': 30,       
            'healthcheck': 9300]],
-	['CHROME': [
-	   'image': 'stratio/selenium-chrome:48',
-           'volumes': [
-		 '/dev/shm:/dev/shm'],
-           'env': [
-		 'SELENIUM_GRID=selenium.cd','ID=%%JUID']]],
+        ['CHROME': [
+            'image': 'stratio/docker-selenium-chrome:0.1.0',
```
Comment (witokondoria) [inline Jenkinsfile]: Why is this image not following the selenium-browser:browser_version naming?

**Comment by gbecares** [comment]
That image is generated by this repo: https://github.com/Stratio/docker-selenium-chrome.

The tag can be changed if necessary, there is not problem.

If we want to upgrade Selenium to 3.4.0 some dependencies as Guava, Cassandra driver and Curator have to be upgraded too.

**Comment by witokondoria** [comment]
I cant see why some (driver) dependencies must be upgraded.

**Pull request closed without merging**